.github/workflows/build-ea.yml

```diff
@@ -9,15 +9,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        java: [ '18-ea' ]
         os: [ubuntu-latest]
     steps:
     - name: Checkout Sources
       uses: actions/checkout@v1
     - name: Setup Java
-      uses: oracle-actions/setup-java@v1
+      uses: actions/setup-java@v2
       with:
-        website: jdk.java.net
-        release: EA
+        distribution: 'temurin'
+        java-version: ${{ matrix.java }}
```

Comment:
Just one thing is that JDK 18 is not EA anymore, only JDK 19 and 20 are. What I liked about the oracle-actions is that it used the latest EA available, meanwhile with Temurin we have to check out when they will publish both JDK 19 and 20 EA versions